eva-pipeline/src/main/java/embl/ebi/variation/eva/pipeline/annotation/generateInput/VariantAnnotationItemProcessor.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2015-2016 EMBL - European Bioinformatics Institute
```

Comment:
Copyright should be 2016 only. @poggio84, @jmmut, do you know if it is acceptable to include the copyright declaration along with the license, to avoid adding all these headers?